CI/install/AppDir/AppRun

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# libobs.so.0 loads its resources relative to the user's current working directory (cwd).
+# Hence we need to cd to usr/ inside the AppDir before executing the application.
+# A proper fix would be to make libobs.so.0 load its resources from a path relative to itself.
+# But seemingly OBS on the Mac works the same, so we can live with this for now.
```

Comment:
OSX now points to the paths in the bundle in various places, but yeah that was the old behavior until catalina broke it
https://github.com/obsproject/obs-studio/pull/2125/files#diff-5284387c0b6ea42e30931b42c288edecR39

Reply:
the environment variable `OBS_DATA_PATH` can be used to point the resources path.